src/components/table/makeSelectable.js

```diff
@@ -326,6 +471,61 @@ function makeSelectable(BaseTable) {
             this.onSelect(selectedItems.add(data[focusedIndex]), newFocusedIndex);
         };
 
+        isContinuation = (selectedItemIndecies, sourceIndex, targetIndex) => {
+            if (sourceIndex < targetIndex && selectedItemIndecies.has(sourceIndex - 1)) {
+                return true;
+            }
+            if (targetIndex < sourceIndex && selectedItemIndecies.has(sourceIndex + 1)) {
+                return true;
+            }
+            return false;
+        };
+
+        handleShiftKeyDownForGrid = newFocusedIndex => {
+            const { data, loadedData, selectedItems } = this.getProcessedProps();
+            const { focusedIndex } = this.state;
+
+            const dataSize = data.length;
+            const targetIndex = newFocusedIndex < 0 ? 0 : Math.min(newFocusedIndex, dataSize - 1);
+            const isSourceSelected = selectedItems.has(data[focusedIndex]);
+            const isTargetSelected = selectedItems.has(data[targetIndex]);
+
+            // if data is not loaded, we don't want it to be able to be selected
```

Comment:
Thanks for the helpful comment here. What does it mean when the data is not loaded? Is that ghosted item?

Reply:
Yeah in the case of `FilePage` it is a ghosted item. i didn't want to put any reference of ghosted items in here though since this is BUIE and this HOC is supposed to be reusable